frontend/src/container/BillingContainer/BillingContainer.styles.scss

```diff
@@ -50,6 +50,13 @@
 			align-items: center;
 		}
 	}
+
+	.billing-update-note {
+		text-align: 'left';
+		font-size: '13px';
+		color: '#dcdcdc';
```

Comment:
you would need to add light mode styles as well

Reply:
Done